packages/ui/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { RadioGroup as ChakraRadioGroup, RadioGroupProps as ChakraRadioGroupProps } from '@chakra-ui/react';
+
+import { Stack, StackProps } from '../Stack';
+
+type RadioGroupSize = 'md' | 'lg';
+
+export interface RadioGroupProps extends Omit<StackProps, 'onChange'> {
+  size?: RadioGroupSize;
```

Comment:
💭 Maybe we can import `RadioSize` from the `Radio` component and use it here. This way it would be always guaranteed that RadioGroup handles all possible sizes of the `Radio`.